sound/soc/sof/topology.c

```diff
@@ -3005,6 +3048,7 @@ static void sof_complete(struct snd_soc_component *scomp)
 			break;
 		}
 	}
+	snd_sof_get_kcontrol_val(sdev);
```

Comment:
Maybe add a comment here letting the readers know that we are caching kcontrol values now in order to use them later and not do calls to DSP everytime.

Reply:
@dbaluta done. I also rename the function.